Classes/TYPO3/Imagine/ImagineFactory.php

```diff
@@ -20,32 +20,34 @@
  */
 class ImagineFactory extends AbstractImagineFactory
 {
-    /**
+	/**
      * @Flow\Inject
```

Comment:
this one should get fixed too right? not sure what's going on with the indentation, seems mixed up?